ui/inspectors/tunable.reel/tunable.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @module ui/tunable.reel
+ */
+var Component = require("montage/ui/component").Component,
+    Model = require("core/model/model").Model,
+    TunableType = require("core/model/enumerations/tunable-type").TunableType;
+
+/**
+ * @class Tunable
+ * @extends Component
+ */
+exports.Tunable = Component.specialize(/** @lends Tunable# */ {
+    typeOPTIONS: {
+        value: null
+    },
+
+    enterDocument: {
+        value: function(isFirstTime) {
+            var self = this,
+                tunableType = TunableType;
+            if (isFirstTime) {
+                this.typeOPTIONS = [];
+                for (var i = 0; i < tunableType.members.length; i++) {
+                    this.typeOPTIONS.push({label: tunableType.members[i], value: tunableType[tunableType.members[i]]});
```

Comment:
The select-option-converter is made to do that conversion for you.
Just put the content of `TunableType.members` in `this.typeOptions` and instantiate the select-option-converter like this in the serialization: 
```
"tunableTypeOptionConverter": {
    "prototype": "core/converter/select-option-converter",
    "properties": {
        "labelPropertyName": "this",
        "valuePropertyName": "this"
},
```